include/awkward/array/NumpyArray.h

```diff
@@ -793,27 +799,38 @@ namespace awkward {
                                            bool ascending,
                                            bool stable) const;
 
-      template<typename T>
-      std::tuple<const std::shared_ptr<void>, const int64_t> array_sort(const T* data,
+    template<typename T>
+    const std::shared_ptr<void> array_sort(const T* data,
+                                           int64_t length,
+                                           const Index64& starts,
+                                           const Index64& parents,
+                                           int64_t outlength,
+                                           bool ascending,
+                                           bool stable) const;
+    template<typename T>
+    const std::shared_ptr<void> array_unique(const T* data,
```

Comment:
@jpivarski - prompted by static analyser, I separated `sort` from `is_unique` operation. The change is purely technical.

Reply:
ok